frontend/src/views/DiagnosticTunnel/WasteMeasureSteps/index.vue

```diff
@@ -1,9 +1,227 @@
+            </v-radio-group>
+          </fieldset>
+        </v-col>
+        <v-col cols="12" sm="6"></v-col>
+      </v-row>
+    </div>
+    <div v-else-if="stepUrlSlug === 'autres'">
+      <v-row>
+        <v-col cols="12" sm="9" md="7">
+          <fieldset>
+            <legend class="my-3">Autres commentaires</legend>
+            <p class="fr-text-xs mt-1 mb-3">
+              Optionnel : toute précision que vous souhaiteriez apporter sur votre situation et/ou sur vos actions mises
+              en place pour lutter contre le gaspillage alimentaire
+            </p>
+            <DsfrTextarea v-model="payload.otherWasteComments" rows="3" class="mt-6" />
+          </fieldset>
+        </v-col>
+      </v-row>
+    </div>
+    <div v-else-if="stepUrlSlug === 'expérimentation'">
+      <h3 class="text-h6 font-weight-bold mb-4">
+        Expérimentation réservation de repas
+      </h3>
+
+      <p class="body-2">
+        Vous souhaitez réduire le gaspillage alimentaire dans votre établissement et générer des économies :
+        <span class="font-weight-bold">la réservation de repas peut être une solution !</span>
+      </p>
+      <p class="body-2">
+        Pour évaluer ses effets sur le gaspillage alimentaire, la satisfaction de vos convives et le taux de
+        fréquentation de votre établissement, nous vous proposons de participer à une expérimentation prévue par la loi
+        climat et résilience.
+      </p>
+      <p class="body-2">
+        Votre candidature à cette expérimentation vous permettra de mettre en place une démarche d’évaluation dont les
+        résultats permettront de saisir le potentiel de la solution de réservation de repas.
+      </p>
+      <p class="body-2 font-weight-bold">
+        Vous avez déjà mis en place une solution de réservation de repas ou souhaitez en adopter une ? Vous pouvez vous
+        inscrire dès maintenant !
+      </p>
+      <p class="body-2">
+        Vous serez amenés à répondre à des questions sur votre structure et la solution de réservation que vous aurez
+        mise en place, ainsi qu’à transmettre des données relatives aux évaluations du gaspillage alimentaire, du taux
+        de fréquentation et de la satisfaction des usagers sur une période de six mois.
+      </p>
+      <p class="body-2">Les inscriptions sont ouvertes jusqu’au 1er juillet 2023.</p>
+      <p class="body-2">
+        Les informations relatives aux conditions de mise en œuvre de l’expérimentation sont précisées dans
+        <a href="/static/documents/Guide_pratique_XP_RESERVATION.pdf" target="_blank">le guide pratique</a>
+        .
+      </p>
+      <v-checkbox v-model="canteen.reservationExpeParticipant" @change="onExpeCheckboxChange">
+        <template v-slot:label>
+          <span class="body-2 grey--text text--darken-3">
+            Je suis volontaire pour participer à l’expérimentation.
```

Comment:
Weird bug - when I click "Mettre à jour mes données" this text disappears
![Screenshot 2023-11-13 at 18-23-50 ma cantine](https://github.com/betagouv/ma-cantine/assets/9282816/070a556a-ea74-47db-ada3-a78b033e2277)

Reply:
Ce bug est seulement en dev lors de l'utilisation du hot-reload. Je l'ai eu aussi avec l'ancien formulaire